docs/proposals/2019-05-13_Tags_Subtype.md

```diff
@@ -0,0 +1,30 @@
+# Problem
+
+Users want to be able to add tags to stories without using the Tags API and they should be searchable by subtype and able to be differentiated between different use cases, such as manual vs. automatic
+
+# Proposal
+
+We propose that the necessary subtype field in tags be included in the ANS schema so that users can search on it.
+
+## Example
+```json
+{
+  "_id": "ABCDEFGHIJKLMNOPQRSTUVWXYZ",
+  "type": "story",
+  "version": "0.10.1",
+
+  "taxonomy": {
+    "tags": {
```

Comment:
Should this be `tag` or an array?

Reply:
according to the schema, it's an object called tags

Reply:
and i just remembered that elasticsearch mappings consider elements and arrays of that element the same type. arrays is right